sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -850,8 +850,8 @@ public async Task AnalyzeDocumentPopulatesExtractedInvoiceJpg(bool useStream)
             Assert.AreEqual("123 Ship St", shippingAddress.StreetAddress);
 
             Assert.AreEqual("Microsoft Delivery", document.Fields["ShippingAddressRecipient"].Value.AsString());
-            ValidateCurrencyValue(document.Fields["SubTotal"].Value.AsCurrency(), 100.00, "$", "USD");
-            ValidateCurrencyValue(document.Fields["TotalTax"].Value.AsCurrency(), 10.00, "$", "USD");
+            ValidateCurrencyValue(document.Fields["SubTotal"].Value.AsCurrency(), 100.00, "$", "CAD");
+            ValidateCurrencyValue(document.Fields["TotalTax"].Value.AsCurrency(), 10.00, "$", "CAD");
```

Comment:
Behavior change on the service side after updating their prebuilt models. Not the first and not the last time this happens.